modules/templates/helper.go

```diff
@@ -85,6 +86,21 @@ func NewFuncMap() []template.FuncMap {
 		"AllowedReactions": func() []string {
 			return setting.UI.Reactions
 		},
+		"ReactionToEmoji": func(reaction string) template.HTML {
```

Comment:
This should also probably be moved to function like `RenderEmoji`

Reply:
They are slightly different -- ReactionToEmoji just returns just the emoji character for use in the existing reactions template and RenderEmoji returns different html with full <span class=emoji> etc...

Some of these seem similar but there are slightly different use cases for them

Reply:
I also have two distinct functions in JS, one that returns HTML and one that returns a unicode string. Maybe rename the helpers for clarity, `RenderEmojiString`, `RenderEmojiHTML`.

Reply:
One converts a 'reaction' as defined in the "REACTIONS" config setting to an emoji character without any span tag etc... and the other renders a full span tag/string for use in almost every other place in templates. Both those functions return template.HTML and not strings. 

There is a third one that returns a plain string and it is RenderEmojiPlain which is needed for page title and email subject that needs to just be a string.

They all use the same underlying functions from emoji.go just return slightly different things as needed.

Reply:
I did not mean that they are the same, I meant that you move this function below and leave only function name here

Reply:
Ah OK I see what you mean sorry I was confused. Fixed : )